<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellerForm</class>
 <widget class="QWidget" name="SellerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

*{
	font: normal 20px “微软雅黑”;
}

#pushButton{
QPushButton

{

color: rgb(255, 255, 255);
    border:none;
    padding: 5px;
    outline: none;
}

QPushButton:pressed
{
    padding-top: -15px;
    padding-bottom: -17px;
}


}
#SellerForm{
	background-color: #818181;
}

QLineEdit{

    border: 1px solid gray;

    padding: 1px 2px 1px 2px;

}

QLineEdit:hover{

border:1px solid blue;

background-color: #818181;

}

</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/images/xiugai.png</normaloff>:/images/xiugai.png</iconset>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*
tabelwidget*/
QTableWidget{
color:#DCDCDC;
background:#444444;
border:1px solid #242424;
alternate-background-color:#525252;/*交错颜色*/
gridline-color:#242424;
}

/*选中item*/
QTableWidget::item:selected{
color:#DCDCDC;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #484848,stop:1 #383838);
}

/*
悬浮item*/
QTableWidget::item:hover{
background:#5B5B5B;
}
/*表头*/
QHeaderView::section{
text-align:center;
background:#5E5E5E;
padding:3px;
margin:0px;
color:#DCDCDC;
border:1px solid #242424;
border-left-width:0;
}



/*表右侧的滑条*/
QScrollBar:vertical{
background:#484848;
padding:0px;
border-radius:6px;
max-width:12px;
}

/*滑块*/
QScrollBar::handle:vertical{
background:#CCCCCC;
}
/*
滑块悬浮，按下*/
QScrollBar::handle:hover:vertical,QScrollBar::handle:pressed:vertical{
background:#A7A7A7;
}
/*
滑块已经划过的区域*/
QScrollBar::sub-page:vertical{
background:444444;
}

/*
滑块还没有划过的区域*/
QScrollBar::add-page:vertical{
background:5B5B5B;
}

/*页面下移的按钮*/
QScrollBar::add-line:vertical{
background:none;
}
/*页面上移的按钮*/
QScrollBar::sub-line:vertical{
background:none;
}
</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>130</y>
     <width>212</width>
     <height>103</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>iD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_id"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>名字</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_name"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>性别</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_sex"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
